### ISSUE TITLE
Diff +/- are not keyboard/screen reader accessible

### ISSUE THREAD

**Issue by jayspadie** [Microsoft/monaco-editor#411]
monaco-editor version: 0.8.3
Browser: Any
OS: Windows 10

During Accessibility testing, it was found that users cannot get diff information with a screen reader while using the up/down arrow keys to navigate through lines of code in Monaco. It correctly reads the code text, but there is no indication whether it was an existing line, an added line (+), or a removed line (-).

I'm not sure how it fits with the current design, but perhaps including the "+" or "-" in the text read would be a reasonable solution if possible.

Shown below is what is read using NVDA while arrowing down, though other screen readers are similar.

![image](https://cloud.githubusercontent.com/assets/5246869/24817376/42212370-1baa-11e7-815b-a6c476c325d0.png)

**Comment by chrisdias** [comment]
dupe of https://github.com/Microsoft/vscode/issues/17263
